docs/release.rst

```diff
@@ -1,23 +1,16 @@
 Release notes
 =============
 
-1.0.0-beta-10
-------------
+1.0.0 (2024-12)
```

Comment:
If we release it today, we would add the day (2024-12-20)

Reply:
Yes, will do